### PR TITLE
WIP: Add documentation about Puhti web interface interactive shell

### PR DESCRIPTION
**Pull request by robinkar**
Do we need some documentation about tmux here?

### CODE REVIEW

File: docs/computing/webinterface/index.md [hunk 1]
```diff
@@ -36,7 +36,12 @@ From here you can browse your files on Puhti, start a shell, view running jobs a
 
 ### Shell
 
-The shell app can be found at the bottom of the page or on the top navbar under the _Tools_ section. This will launch a normal linux shell on one of the Puhti login nodes. Note that the same rules apply here as during a normal ssh session. **Login node are only for light pre/post processing** (see [Usage policy](/computing/overview/#usage-policy)).
+The shell apps can be found at the bottom of the page or on the top navbar under the _Tools_ section.
+There are two different shells.
+
+The login shell launches a normal linux shell on one of the Puhti login nodes. Any command that is running when the login shell browser tab is closed will stop. Note that the same rules apply here as during a normal ssh session. **Login node are only for light pre/post processing** (see [Usage policy](/computing/overview/#usage-policy)).
```
Comment (CSC-swesters) [inline docs/computing/webinterface/index.md:42]: - "Login node**s** are only ..."
- "...for light pre/postprocessing" (no space)
- If possible, please keep the line width below 120 characters or so. It makes the markdown files easier to read in an editor or a console :slightly_smiling_face: 
- Should the name of the tool "login shell" be highlighted somehow, so that readers can differ between names and concepts? E.g. "login shell" vs. "linux shell" vs. "interactive shell" vs. "persistent shell" feels a bit hard to follow.

Comment (robinkar) [reply docs/computing/webinterface/index.md]: Fixed those things.

**Comment by robinkar** [comment]
Moved to #1115

File: docs/computing/webinterface/index.md (hunk 1 above)
Comment (galfthan) [reply docs/computing/webinterface/index.md]: That is true, especially since login shell has a meaning in linux. Should the apps actually be called "Login node shell" and "Compute node shell"  to make the naming less confusing.
